Update quota labels for client throughput metric (#1785)
redpanda-data/documentation: docs/upgrade/migrate/kubernetes/helm-to-operator.mdx

    <link rel="canonical" href="https://docs.redpanda.com/docs/migrate/kubernetes/helm-to-operator/" />
</head>

If you are using the Redpanda Helm chart, you can migrate to the Redpanda Operator and use it to manage your Helm deployment.

The Redpanda Operator complements the Redpanda Helm chart by extending Kubernetes with custom resource definitions (CRDs). These CRDs allow Redpanda clusters to be represented as native Kubernetes resources. This abstraction makes it simpler to manage the lifecycle of Redpanda deployments, as the Redpanda Operator takes the specifications declared in the Redpanda custom resources and passes them to Helm for execution.

When using the Redpanda Operator, configurations defined in the custom resources are passed to Helm, which then deploys the Redpanda cluster using the Redpanda Helm chart.

:::caution
Redpanda Operator is in beta. Do not migrate existing production clusters to the Redpanda Operator.
:::

## Prerequisites

> Before you can migrate to the Redpanda Operator, you need the following:

Reviewer: ```suggestion
Before migrating to the Redpanda Operator, you must have:
```
